### ISSUE TITLE
Basic README.md with instructions to build the example.

### ISSUE THREAD

**Issue by glennpratt** [dylanede/cef-rs#5]
I assume I need the Chromium Embedded Framework, but I'm not sure where to put it or what prep is needed. :smile: 

```
cargo run --example simple
   Compiling cef v0.0.1 (file:///Users/glenn.pratt/Code/cef-rs)
error: linking with `cc` failed: exit code: 1

<snip>
ld: framework not found Chromium Embedded Framework
<snip>
```

**Comment by dylanede** [comment]
Yeah, sorry. This is still very much a WIP, and I was planning on adding a readme and making a post on reddit after a bit more basic functionality is available. It seems that it is getting a bit of interest before that happens though, so I might add a readme soon instead.

**Comment by Aworldc** [comment]
Yeah. A basic readme would be great as i'm planning to use this.
